libs/community/langchain_community/callbacks/openai_info.py

```diff
@@ -15,6 +15,7 @@
     "gpt-4-32k-0613": 0.06,
     "gpt-4-vision-preview": 0.01,
     "gpt-4-1106-preview": 0.01,
+    "gpt-4-0125-preview": 0.01,
```

Comment:
Can you also add `gpt-4-turbo-preview`, which always points to the latest `gpt-4-turbo` model